mobile-app/src/components/SearchBar.tsx

```diff
@@ -20,21 +24,32 @@ const styles = StyleSheet.create({
 
 // Component
 
-interface Props {
-  text: string;
-}
+//Redux goodness.
 
-export const SearchBar: React.FC<Props> = (props: Props) => {
-  const [query, setQuery] = useState('');
+const mapState = (state: RootState) => ({
+  search: state.artwork.searchQuery,
```

Comment:
It might be more clear if this were named `searchQuery` instead of `search`, but at the end of the day, I don't mind either way. What do y'all think?